### DISCUSSION TITLE
Cool tool, slick, nicely implemented, brain dead easy to run, but missing something...

### DISCUSSION THREAD

**Discussion by mikepoz59** [provectus/kafka-ui#2558] [Ideas]
Hey all, like what I see so far.

I grabbed the latest docker image, ran it in Docker Desktop adding the port, the cluster name, and the bootstrap server URL, connected immediately and gave me a lot of great information about my Kafka service running in my workspace K8S cluster, so yay,, but the thing that seems to be missing because of our needs are the inbound and outbound message rates.  

We need this because we get literally tens of thousands of message in under a minute across a number of topics, and we need to know the average, and peak inbound rates, as well as the outbound rates so we can estimate how long it will take the remaining messages to be processed by our services that listen to the Kafka Service.

Am I missing this in the UI?  If not, are there plans to add this for those of us who need this information?

Thanks!

**Comment by Haarolean** [comment]
Hey, thanks for the feedback.
We're planning to implement it within #942
